charts/druid/templates/druid-clusterrole.yaml

```diff
@@ -0,0 +1,99 @@
+---
```

Comment:
Can you please also add permission for `PodDisruptionBudget`? (https://github.com/gardener/etcd-druid/pull/250)

Reply:
In addition, Druid needs the following permissions:
- list/watch/get all `Lease`s
- get/list/watch/create/patch/update `Job`s
- get/list/watch/update/patch `etcdcopybackupstasks`
- get/create/update/patch `etcdcopybackupstasks/status`